packages/oak-prosemirror-editor/lib/index.styl

```diff
@@ -0,0 +1,5 @@
+@import 'prosemirror-view/style/prosemirror.css'
```

Comment:
is there any minified version of the styles? is it heavy? if it is, can we only load the parts we use from this?

Reply:
its litterally 51 lines.... unminified

https://github.com/ProseMirror/prosemirror-view/blob/master/style/prosemirror.css

Reply:
I can move it on our css file instead ?

Reply:
nop it will be minified anyway by postcss my question was dumb